src/client/App.scss

```diff
@@ -47,7 +47,7 @@ h6 {
 }
 
 h2 {
-  margin-bottom: 2rem;
```

Comment:
hm, so this might change more than you intended, because it affects all h2 elements everywhere including the guide. unless you reviewed all the h2s everywhere including the guide? which h2 elements would you like to change?

Reply:
hmm I'd like to make this blanket update across the Retro Cert app, not the Guide...